Cabal/Cabal.cabal

```diff
@@ -257,6 +257,7 @@ library
     Distribution.Utils.NubList
```

Comment:
[[GH doesn't let me review the commit message so this is here instead]]

> introduces the buildinfo-components-only flag

I don't see this being introduced here, seems like the commit message has fallen out of sync with the diff?